src/ocean/text/convert/Formatter.d

```diff
@@ -911,3 +924,95 @@ private struct FormatInfo
 
     public Flags flags;
 }
+
+/*******************************************************************************
+
+    Retrieves the members of an enumerated type `enum E`.
+
+    Params:
+        E = An enumerated type. `E` may have duplicated values.
+
+    Returns:
+        Static tuple composed of the members of the enumerated type `E`.
+        The members are arranged in the same order as declared in `E`.
+        The name of the enum can be found by querying the compiler for the
+        name of the identifier, i.e. `__traits(identifier, EnumAllMembers!MyEnum[i])`.
+
+*******************************************************************************/
+
+template EnumAllMembers(E)
+if (is(E == enum))
+{
+    import ocean.meta.AliasSeq;
+
+    template MemberIdentifier(string ident)
+    {
+        static if (ident == "MakeAlias")
```

Comment:
What is this ?

Reply:
It means when the ident happens to be "MakeAlias."

Reply:
I'll remove this.